### ISSUE TITLE
Using SOM and deep learning to predict credit card frauds.

### ISSUE THREAD

**Issue by Rohit-Sharma-RS** [mmujtabah/CodingChallenges#12]
Credit card frauds are quite common lately, this is a small contribution from my side in this direction to create a SOM which protects users from credit card frauds even with minimal data.

**Comment by Rohit-Sharma-RS** [comment]
Hi @mmujtabah I am currently working on this can you assign this issue to be under the hacktoberfest accepted label?
Thanks.

**Comment by alizaincodes** [comment]
@Rohit-Sharma-RS I appreciate your idea. By adding this challenge to the repository, will it be a friendly challenge for beginners?

Will Neural Networks and Self-Organizing Maps be an easy challenge for beginners who are new in programming?

**Comment by Rohit-Sharma-RS** [comment]
Hi @alizaincodes 
No this might not be a beginner friendly project unless there is some understanding of machine learning(especially K Means clustering algorithm) and at least basic knowledge on artificial neural networks.

**Comment by alizaincodes** [comment]
Hi @Rohit-Sharma-RS 
Okay now I understand that this challenge is made for those who are new in AI and ML. 
By the way, how much data you need for training?

**Comment by Rohit-Sharma-RS** [comment]
The dataset has around 1000 rows planning to train them over 50-70 epochs

**Comment by mmujtabah** [comment]
> Hi @alizaincodes No this might not be a beginner friendly project unless there is some understanding of machine learning(especially K Means clustering algorithm) and at least basic knowledge on artificial neural networks.

We could introduce separate sections for beginners and advanced users, placing this project in the advanced section. That way, those with foundational knowledge can still explore and contribute at their own pace.

Great idea, by the way! I’ll definitely look forward to contributing as soon as I find some free time.

**Comment by alizaincodes** [comment]
@mmujtabah This would be nice as users have to go to their respective folders(beginner or advanced). This will make it easy to find and categorize challenges.

**Comment by mmujtabah** [comment]
> @mmujtabah This would be nice as users have to go to their respective folders(beginner or advanced). This will make it easy to find and categorize challenges.

Absolutely! Feel free to add an advanced section after all, it’s an open-source repo, and every contribution counts. Your efforts will definitely make it more organized and easier for users to find relevant challenges.

**Comment by Rohit-Sharma-RS** [comment]
The project was added please review it.
I created a separate advanced section to add the project into.
